tests/queries/0_stateless/03285_analyzer_array_join_nested.reference

```diff
@@ -64,3 +64,8 @@ QUERY id: 0
 SELECT tupleElement(__array_join_exp_1, \'names\') AS `metric.names`
 FROM default.hourly AS __table2
 ARRAY JOIN nested(_CAST([\'names\', \'values\'], \'Array(String)\'), __table2.`metric.names`, __table2.`metric.values`) AS __array_join_exp_1
+-- { echoOn }
+
+select y, y.b.first, y.b.second from tab array join x as y;
+('a1',[1,2,3],['b1','b2','b3'])	[1,2,3]	['b1','b2','b3']
+('a2',[4,5,6],['b4','b5','b6'])	[4,5,6]	['b4','b5','b6']
```

Comment:
Probably, it should be controlled by a separate setting to avoid problems with backwards compatibility.